### PR TITLE
added Filters on Home Page- sort by distance, service_category

### PR DESCRIPTION
**Pull request by maheshkurma**
Added New Section - Advanced Filtering in Home Page 

When you search events with zip, now we have the ability to filter those events by Distance and Service Category Type.(for now).
Screen Shot 2021-01-29 at 9 45 52 AM

### CODE REVIEW

**Comment by sarath-yerra** [comment]
UI Looks Good Filtering Events by Service Catogory

**Comment by sarath-yerra** [comment]
We need to wait Until we Merge Back end

**Review by sarath-yerra — APPROVED** [review]

**Comment by maheshkurma** [comment]
backend PR merged.so merging this into master now.

**Pull request merged**
